x-pack/plugins/cloud_security_posture/public/pages/rules/index.tsx

```diff
@@ -41,6 +52,56 @@ const getRulesBreadcrumbs = (
   return breadCrumbs;
 };
 
+const assertPolicyTemplate = (
+  policyTemplate: unknown
+): policyTemplate is keyof CloudPostureIntegrations => {
+  if (typeof policyTemplate !== 'string') return false;
+
+  return cloudPostureIntegrations.hasOwnProperty(policyTemplate);
+};
+
+const getRulesSharedValues = (
+  packageInfo: PackagePolicy,
+  agentInfo: AgentPolicy
+): EuiDescriptionListProps['listItems'] => {
+  const enabledPackage = packageInfo.inputs?.find((input) => input.enabled);
+  if (!enabledPackage || !assertPolicyTemplate(enabledPackage.policy_template)) return;
+
+  const integration = cloudPostureIntegrations[enabledPackage.policy_template];
+  if (!integration) return;
+
+  const enabledIntegrationOption = integration.options.find(
+    (option) => option.type === enabledPackage.type
+  );
+
+  return [
+    {
+      title: i18n.translate('xpack.csp.rules.rulesPageSharedValues.integrationTitle', {
+        defaultMessage: 'Integration',
+      }),
+      description: integration.shortName,
+    },
+    !!enabledIntegrationOption?.name && {
+      title: i18n.translate('xpack.csp.rules.rulesPageSharedValues.typeTitle', {
+        defaultMessage: 'Type',
+      }),
+      description: enabledIntegrationOption.name,
+    },
+    !!enabledIntegrationOption?.benchmark && {
+      title: i18n.translate('xpack.csp.rules.rulesPageSharedValues.benchmarkTitle', {
+        defaultMessage: 'Benchmark',
+      }),
+      description: enabledIntegrationOption.benchmark,
+    },
+    {
+      title: i18n.translate('xpack.csp.rules.rulesPageSharedValues.agentPolicyTitle', {
+        defaultMessage: 'Agent Policy',
+      }),
+      description: agentInfo.name,
+    },
```

Comment:
@tinnytintin10 i've left the agent policy name, wasn't sure if it was purposely removed or just forgotten in the figmas, feels like important information. 
wdyt?